src/exchange/group/accounts/group-accounts.controller.js

```diff
@@ -28,94 +28,92 @@ angular.module('Module.exchange.controllers').controller(
         managersList: [],
         membersList: [],
       };
-
-      $scope.getAccounts = (count, offset) => this.getAccounts(count, offset);
-      $scope.updateAccounts = () => this.updateAccounts();
-      $scope.getAccountsList = () => this.accountsList;
-      $scope.getLoading = () => this.loading;
     }
 
-    resetSearch() {
-      this.search.value = null;
-      this.services.$scope.$broadcast('paginationServerSide.loadPage', 1, 'accountsByGroupTable');
-    }
+    updateManagersList(newManagerValue, account) {
+      const bufferedAccount = _.find(
+        this.accountsListBuffer.list.results,
+        bufferedAcc => bufferedAcc.id === account.id,
+      );
```

Comment:
You can get rid of lodash :)
```suggestion
      const bufferedAccount = this.accountsListBuffer.list.result.find(({ id }) => id === account.id);
```